src/hello-n.py

```diff
@@ -1,6 +1,9 @@
 
 # this asks the user for a number and turns the answer
 # into an integer
-n = int(input("How many times should I loop? "))
-
+n = 11
```

Comment:
This looks like it is working, but the test is failing because you hardwired `n` to be 11 instead of using the `input()` code.